src/safe_apps/tests/test_views.py

```diff
@@ -178,6 +219,126 @@ def test_apps_returned_on_same_key_pair(self) -> None:
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.json(), json_response)
 
+    def test_apps_returned_on_non_existent_client_url(self) -> None:
+        safe_app = SafeAppFactory.create()
+        url = reverse("v1:safe-apps:list") + f'{"?client_url=non_existent_host"}'
```

Comment:
Why the query parameter here? We don't do any custom handling with it right?

Reply:
It's there to check how the API reacts if you pass an unknown client URL. The `client_url` param is used to return all the apps that work on passed URL

Reply:
Sorry you are right. I see that you added it on the view side 👍